src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -85,6 +88,35 @@ const ActiveState = ({
                 const isFocused = item === activeEntry;
                 const refValue = isFocused ? activeFeedItemRef : undefined;
                 const itemFileVersionId = getProp(item, 'file_version.id');
+                const isCurrentVersion = currentFileVersionId === itemFileVersionId;
+
+                if (hasReplies && (item.type === 'comment' || item.type === 'annotation')) {
+                    return (
+                        <ActivityItem
+                            key={item.type + item.id}
+                            className="bcs-activity-thread"
+                            data-testid="activity-thread"
+                            isFocused={isFocused}
+                            ref={refValue}
+                        >
+                            <ActivityThread
+                                currentFileVersionId={currentFileVersionId}
+                                getAvatarUrl={getAvatarUrl}
+                                isCurrentVersion={isCurrentVersion}
+                                item={item}
+                                onAnnotationDelete={onAnnotationDelete}
+                                onAnnotationEdit={onAnnotationEdit}
+                                onAnnotationSelect={onAnnotationSelect}
+                                // onAnnotationReplyCreate={() => {}} // TODO will be implemented in next PR
```

Comment:
We generally want to avoid commented code, even if it's temporary. Can we omit these lines?